server/src/main/java/org/opensearch/common/cli/package-info.java

```diff
@@ -9,4 +9,4 @@
 /**
  * The command line interface module.
  */
-package org.opensearch.cli;
+package org.opensearch.common.cli;
```

Comment:
Just a question @msfroh  @prudhvigodithi what do you think folks the right name for the package  should be?

So we have:

```
org.opensearch.cli
org.opensearch.cli.keystore
```

It looks to me we should have:

```
org.opensearch.cli.common
```

What do you think?

Reply:
@reta It makes sense that all CLI classes use the `o.o.cli` prefix. I just don't like the `:server` module would "own" the generic sounding `o.o.cli.common` namespace. How about using `o.o.cli.server` since these are the CLI-related classes that have a dependency on `:server` components?

Reply:
We should go with `org.opensearch.common.cli`, as it indicates the location of the common CLI classes for the `:server` module (from other packages inside :server common).

Reply:
I would personally like to eventually get rid of any usage of `o.o.common` in the `:server` module, as I would expect `o.o.common` to be used by `:libs:opensearch-common`, so that makes `org.opensearch.common.cli` my least preferred option.